php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -544,23 +553,6 @@ class NDB_Menu_Filter extends NDB_Menu
             }
         }
 
-        if (isset($this->searchKeyword)
```

Comment:
I did a `git grep searchKeyword` and no modules in LORIS are making use of this.